### ISSUE TITLE
Question regarding Sigcheck

### ISSUE THREAD

**Issue by coleJ98** [ukncsc/lme#5]
Hi, Thanks for putting this together! Its a great resource and easily understandable. I wanted to ask a question regarding sigcheck.

I was wondering why you have included the sigcheck tool along with Sysmon? Since sysmon provides you with the file hash anyway, is there a need for Sigcheck? Or is there another reason/use for this tool? Please let me know. Thanks!

**Comment by coleJ98** [comment]
Sorry, I accidentally labelled it as a bug.

**Comment by duncan-ncc** [comment]
Sigcheck is used to check the versions of Sysmon available and deployed. 
We cannot use Sysmon to determine the hash before Sysmon is installed, also without running the centrally stored version of Sysmon we are unable to get the filehash from Sysmon so we use Sigcheck for this.

**Comment by shane-m1** [comment]
In addition, I think we used it to check the version number of the sysmon binary (the one in the binary header). We didn’t want every machine pulling down sysmon once a day to check if it had changed, for network bandwidth and unintentional DDoS reasons. Instead, we tested and sigcheck appears to be using windows networking to just read the file version, so takes minimal bandwidth. If we see the number change, then we pull it down. We can’t do this for the config xml, so pull that down and hash but it’s about 100kb.
